pkg/config/property/urischeme/urischeme.go

```diff
@@ -0,0 +1,113 @@
+package urischeme
+
+import (
+	"fmt"
+	"os"
+	"regexp"
+	"strings"
+
+	"github.com/vincent-petithory/dataurl"
+)
+
+// URIScheme represents a string type that defines a URI scheme.
+// Supported URI schemes are:
+// - /path/to/file (backward compatibility)
+// - file:///path/to/file (RFC 8089)
+// - data:;base64,SGVsbG8sIFdvcmxkIQ== (RFC 2397)
+// - data:,plgd (RFC 2397)
+type URIScheme string
+
+var anyScheme = regexp.MustCompile(`^[a-zA-Z]+:+//`)
+
+// ToURISchemeArray converts a slice of strings to a slice of URIScheme.
+func ToURISchemeArray(v []string) []URIScheme {
+	result := make([]URIScheme, 0, len(v))
+	for _, s := range v {
+		result = append(result, URIScheme(s))
+	}
+	if len(result) == 0 {
+		return nil
+	}
+	return result
+}
+
+// ToFilePathArray returns an array of file paths from an array of URIScheme.
+// If a URIScheme does not have a file path, it is not included in the result.
+func ToFilePathArray(v []URIScheme) []string {
+	result := make([]string, 0, len(v))
+	for _, s := range v {
+		f := s.FilePath()
+		if f != "" {
+			result = append(result, s.FilePath())
+		}
+	}
+	return result
+}
+
+// IsFile returns true if the URIScheme is a file scheme.
+func (p URIScheme) IsFile() bool {
+	if len(p) == 0 {
+		return false
+	}
+	if p.IsData() {
+		return false
+	}
+	if strings.HasPrefix(string(p), "file:///") {
```

Comment:
relative paths are not supported? `file://path/to/file` won't match this prefix but will match `anyScheme`.

Reply:
OK, apparently it's correct, relative paths are not supported by the URI scheme strings